src/compounds/LinkedButton/LinkedButton.jsx

```diff
@@ -31,7 +31,7 @@ const { onClick, ...remainingPropTypes } = Button.propTypes
 LinkedButton.propTypes = {
   buttonProps: PropTypes.shape(remainingPropTypes).isRequired,
   addClass: PropTypes.string,
-  path: PropTypes.exact({
+  path: PropTypes.shape({
```

Comment:
why change this?

Reply:
previously it threw an error with exact when we passed it a path with no query
<img width="645" alt="image" src="https://user-images.githubusercontent.com/73361970/215861242-c1403ac4-272c-419b-83ff-d913f69dfdc4.png">

but but since now we are always passing the pathname and the query it is fine to bring it back

Reply:
i see there is still a linting issue with the LinkedButton, but i maded a separate ticket for it here
https://github.com/scientist-softserv/webstore-component-library/issues/143